src/db/pool_unprepared_statement.cr

```diff
@@ -0,0 +1,21 @@
+module DB
+  # Represents a statement to be executed in any of the connections
+  # of the pool. The statement is not be executed in a non prepared fashion.
+  # The execution of the statement is retried according to the pool configuration.
+  #
+  # See `PoolStatement`
+  class PoolUnpreparedStatement < PoolStatement
+    def initialize(db : Database, query : String)
+      super
+    end
+
+    protected def do_close
+      # unprepared statements do not need to be release in each connection
+    end
+
+    # builds a statement over a real connection
+    private def build_statement
+      @db.pool.checkout.unprepared.build(@query)
```

Comment:
I'd rather call `build_unprepared_statement` directly here, to avoid creating the extra intermediate struct